imsim/sag.py

```diff
@@ -0,0 +1,82 @@
+import galsim
+from galsim.config import ExtraOutputBuilder, RegisterExtraOutput, GetInputObj
+from galsim.config.input import ParseValue, GetAllParams
+import numpy as np
+import batoid
+from .telescope_loader import infer_optic_radii
+
+
+class SagBuilder(ExtraOutputBuilder):
+    """Build surface sag maps.
```

Comment:
I'm sure for the people who know, sag is appropriate jargon.  But at least once, you should define what this word means.